.gitignore

```diff
@@ -49,3 +49,6 @@ coverage.xml
 docs/_build/
 
 tests/welt2000/data/WELT2000.TXT
+
+# Mac
+*.DS_STORE
```

Comment:
I would recommend to use https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer to ignore such files